sdk/clientcore/annotation-processor/src/main/java/io/clientcore/annotation/processor/AnnotationProcessor.java

```diff
@@ -176,8 +176,12 @@ private HttpRequestContext createHttpRequestContext(ExecutableElement requestMet
                 method.addQueryParam(queryParam.value(), param.getSimpleName().toString(),
                     queryParam.multipleQueryParams(), !queryParam.encoded());
             } else if (bodyParam != null) {
-                method.setBody(new HttpRequestContext.Body(bodyParam.value(), param.asType().toString(),
-                    param.getSimpleName().toString()));
+                if (bodyParam.value() == null) {
```

Comment:
Glad to see we're being defensive here, but this should never happen as `value` is required as part of the annotation definition, so anyone who doesn't set one should get a compilation error before this is ever reached.